src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -24,19 +30,124 @@ using neural_net::float_array_map;
 using neural_net::model_backend;
 using neural_net::model_spec;
 using neural_net::shared_float_array;
+using neural_net::weight_initializer;
+using neural_net::zero_weight_initializer;
+
+using padding_type = model_spec::padding_type;
+// anonymous helper sections
 
 }  // namespace
 
 std::unique_ptr<model_spec> drawing_classifier::init_model() const {
   std::unique_ptr<model_spec> result(new model_spec);
+
+  // state is updated through init_train
+  flex_string target = read_state<flex_string>("target");
+  size_t num_classes = read_state<flex_int>("num_classes");
+
+  // feature columns names
+  const flex_list &features_list = read_state<flex_list>("features");
+
+  result->add_channel_concat(
+      "features",
+      std::vector<std::string>(features_list.begin(), features_list.end()));
+
+  weight_initializer initializer = zero_weight_initializer();
+
+  std::string prefix{"drawing"};
+  std::string input_name{"features"};
+  std::string output_name;
+
+  {
+    size_t channels_filter = 16;
+    size_t channels_kernel = 1;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
Also, please use `constexpr` for compile-time constants

Reply:
These are actually variables, not constants.